src/cli/npc.rs

```diff
@@ -1,35 +1,59 @@
-use nispor::get_state;
+use clap::{clap_app, crate_authors, crate_version};
+use nispor::NetState;
 use serde_json;
-use std::env::args;
+use std::process;
 
 fn main() {
-    let argv: Vec<String> = args().collect();
-    match get_state() {
+    let matches = clap_app!(npc =>
+        (version: crate_version!())
+        (author: crate_authors!())
+        (about: "Nispor CLI")
+        (@subcommand link =>
```

Comment:
Can we use `npc eth1` instead of `npc link eth1`?

And assuming there is no interface named as `route`, so `npc route dev eth1` is acceptable.

This might require some hack in clap, but that could save a lot typing time of me.

Reply:
Are you sure? Having 'npc IFNAME' and 'npc route IFNAME' seems a bit inconsistent.

Reply:
It might be tricky for code and document. let me think it through.